README.md

```diff
@@ -25,11 +25,33 @@ It's mighty because it:
 
 ## Getting started
 
-You'll find everything you need to get started in our [docs](docs/toc.md).
+1\. Use [npm](https://docs.npmjs.com/about-npm/) to install stylelint and its [`standard configuration`](https://github.com/stylelint/stylelint-config-standard):
+
+```shell
+npm install -D stylelint stylelint-config-standard
```

Comment:
[IMO] I think that `--save-dev` may be a bit kinder than `-D` for those who are unfamiliar to the npm CLI.

```suggestion
npm install --save-dev stylelint stylelint-config-standard
```